CliClient/locales/et_EE.po

```diff
@@ -0,0 +1,3197 @@
+#, fuzzy
+msgid "Step 1: Open this URL in your browser to authorise the application:"
+msgstr "Samm 1: avage see URL brauseris, et lubada rakendust:"
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:89
+#: /Users/tessus/data/work/joplin/Tools/../ElectronClient/app/gui/DropboxLoginScreen.min.js:59
+#: /Users/tessus/data/work/joplin/Tools/../ReactNativeClient/lib/components/screens/dropbox-login.js:62
+#, fuzzy
+msgid "Step 2: Enter the code provided by Dropbox:"
+msgstr "2. samm: sisestage kood, mis on esitatud Dropbox:"
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:101
+#, fuzzy, javascript-format
+msgid "Not authentified with %s. Please provide any missing credentials."
+msgstr "Pole %s-ga seostatud. Palun esitage puuduv mandaat."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:124
+#, fuzzy
+msgid "Synchronisation is already in progress."
+msgstr "Sünkroniseerimine on juba pooleli."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:129
+#, fuzzy, javascript-format
+msgid ""
+"Lock file is already being hold. If you know that no synchronisation is "
+"taking place, you may delete the lock file at \"%s\" and resume the "
+"operation."
+msgstr ""
+"Lukusta fail on juba kinni. Kui teate, et sünkroonimis ei toimu, võite luku "
+"faili kustutada kohas %s ja jätkata toimingut."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:175
+#, fuzzy, javascript-format
+msgid "Synchronisation target: %s (%s)"
+msgstr "Sünkroonimise sihtkoht: %s ( %s)"
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:177
+#, fuzzy
+msgid "Cannot initialize synchroniser."
+msgstr "Sünkroonijat ei saa lähtestada."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:179
+#, fuzzy
+msgid "Starting synchronisation..."
+msgstr "Sünkroonimise käivitamine..."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:202
+#, fuzzy
+msgid "Downloading resources..."
+msgstr "Ressursside allalaadimine..."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-sync.js:226
+#, fuzzy
+msgid "Cancelling... Please wait."
+msgstr "Tühistamine... Oota."
+
+#: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-tag.js:14
+#, fuzzy
+msgid ""
+"<tag-command> can be \"add\", \"remove\", \"list\", or \"notetags\" to "
+"assign or remove [tag] from [note], to list notes associated with [tag], or "
+"to list tags associated with [note]. The command `tag list` can be used to "
+"list all the tags (use -l for long option)."
+msgstr ""
+"<tag-command>võib olla \"lisa\", \"Eemalda\", \"loend\" või \"notetags\", et "
+"määrata või eemaldada [silt], [märge], loetleda märkmed, mis on seotud "
+"[tag], või loetleda silte seostatud [Note]. Käsu \"tag list\" saab kasutada "
+"loetleda kõik sildid (kasutamine-l pikka võimalust)."
```

Comment:
You cannot translate command names, otherwise they won't be recognized by Joplin when a user enters them.
However, you could add the translation in parenthesis after the command name.

Something like:

```
"<tag-command>võib olla \"add\" (lisa), \"Eemalda\" (remove), ...
```

Reply:
I commit correction of mistakes